src/services/classifier2020.ts

```diff
@@ -0,0 +1,257 @@
+/** @internal */
+namespace ts.classifier.modern {
+
+    /** @internal */
+    export enum TokenEncodingConsts {
+        typeOffset = 8,
+        modifierMask = (1 << typeOffset) - 1
+    }
+
+    /** @internal */
+    export enum TokenType {

```

Comment:
I'd make this a `const` enum if possible, especially since it's internal.

```suggestion
    export const enum TokenType {
```